internal/xds/translator/cluster.go

```diff
@@ -100,9 +102,7 @@ func buildXdsCluster(args *xdsClusterArgs) *clusterv3.Cluster {
 		Name:            args.name,
 		DnsLookupFamily: dnsLookupFamily,
 		CommonLbConfig: &clusterv3.Cluster_CommonLbConfig{
-			LocalityConfigSpecifier: &clusterv3.Cluster_CommonLbConfig_LocalityWeightedLbConfig_{
-				LocalityWeightedLbConfig: &clusterv3.Cluster_CommonLbConfig_LocalityWeightedLbConfig{},
-			},
+			LocalityConfigSpecifier: &clusterv3.Cluster_CommonLbConfig_ZoneAwareLbConfig_{},
```

Comment:
The default LbConfig is being changed but if a backend service doesn't also enable zone aware routing the zone information isn't added to LB Endpoints and as a result routing logic is the same as today.

Reply:
Could you please also add this comments to the code? It would help the people come later to understand why zone aware routing is chosen as default.

Reply:
Since updating with cluster per-DestinationSetting I changed the default back to locality weighted